CONTRIBUTING.md

```diff
@@ -40,6 +51,19 @@ Start Jupyter Notebook in another terminal:
 jupyter lab
 ```
 
+After you have made some changes, recompile with:
+
+```
+conda activate s3contents && make env && jupyter notebook
+```
+
+or use this faster command to copy changes to s3contents installation path:
+
+```bash
+rsync -r --exclude '.git' ./s3contents/ $(echo "$(pip show s3contents | grep Location: | cut -d' ' -f2)/s3contents/") && jupyter notebook
+
```

Comment:
Why is this part needed? If you do `make env` poetry will install the package in edit mode so you should not need this.

Reply:
see below comment, separated this and explained that this is a fallback method.